src/tests/src/libs/antares/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(WITH_YAMLCPP)
 endif()
 
 if(WITH_ANTLR4)
-    add_subdirectory(antlr-interface)
+    add_subdirectory(antlr4-interface)
```

Comment:
why are you doing this?

Reply:
you should rename the lib also and the dir under src/lib/

Reply:
src/tests/src/libs/antares/antlr4-interface/test_antlr_interface.cpp